modules/library/main/src/main/java/org/geotools/util/ColorConverterFactory.java

```diff
@@ -354,7 +354,7 @@ public String toString() {
      * <li>Hex representation of the form <code>#RRGGBB</code> and <code>#RRGGBBAA</code></li>
      * </ul>
      */
-    private static Converter CONVERT_CSS_TO_COLOR = new Converter() {
+    public static Converter CONVERT_CSS_TO_COLOR = new Converter() {
```

Comment:
These two visibility switches seem to be a leftover from a previous attempt? They are not used elsewhere in the patch.